apps/src/applab/firebaseCounters.js

```diff
@@ -0,0 +1,124 @@
+/* global Applab */
+
+import Firebase from 'firebase';
+import { getDatabase } from './firebaseUtils';
+
+const TABLE_ROW_COUNT_LIMIT = 10;
+
+/**
+ * Map representing the maximum number of writes allowed over each time interval.
+ *
+ * @type {Object.<string, number>} Map from rate limit interval in seconds
+ *     to the maximum number of operations allowed during that interval.
+ */
+const RATE_LIMITS = {
+  15: 300,
+  60: 600
+};
```

Comment:
Are these just a client-side version of something that's also enforced by firebase rules?

If so: Is there any better way to keep these constants in sync with those rules - can they be set by some CDO configuration at build-time?

If not: Couldn't a client change these at runtime?  The `const` gets transpiled to a simple `var`, I believe.